src/solver/conv_wino_fury_RxS.cpp

```diff
@@ -0,0 +1,569 @@
+    kernel_file << kernel_postfix;
+
+    if(problem.GetKernelStrideW() == 1)
+    {
+        kernel_name << "_stride1";
+        kernel_file << "_stride1";
+    }
+    else if(problem.GetKernelStrideW() == 2 && !problem.direction.IsBackwardData())
+    {
+        kernel_name << "_stride2";
+        kernel_file << "_stride2";
+    }
+    else // if(problem.GetDilationH() == 2)
+    {
+        kernel_name << "_dilation2";
+        kernel_file << "_dilation2";
+    }
+
+    kernel.kernel_name = kernel_name.str();
+    kernel_file << ".s";
+    kernel.kernel_file = kernel_file.str();
+
+    result.construction_params.push_back(kernel);
+
+    constexpr uint8_t IDENTITY = 0;
+    // todo: support fused version
+    // constexpr uint8_t LEAKY_RELU = 1;
+    // constexpr uint8_t SIGMOID    = 2;
+    // constexpr uint8_t SCALED_TANH = 3;
+    uint8_t activation_mode = IDENTITY;
+
+    uint32_t flags = 0;
+
+    const bool is_forward = problem.direction.IsForward();
+    const bool is_backWrW = problem.direction.IsBackwardWrW();
+    const int group_cnt   = problem.GetGroupCount();
+
+    MemLayout_t d_layout, o_layout, f_layout;
+    if(!is_backWrW)
+    {
+        d_layout = GetGroupConvLayout(GetMemLayout_t(problem.GetInLayout()), true);
+        o_layout = GetGroupConvLayout(GetMemLayout_t(problem.GetOutLayout()), true);
+        f_layout = GetGroupConvLayout(
+            is_forward ? MemLayout_t::NCHW : GetSwappedNCLayout(MemLayout_t::NCHW), false);
+    }
+    else
+    {
+        d_layout =
+            GetGroupConvLayout(GetSwappedNCLayout(GetMemLayout_t(problem.GetInLayout())), true);
+        o_layout =
+            GetGroupConvLayout(GetSwappedNCLayout(GetMemLayout_t(problem.GetOutLayout())), false);
+        f_layout = GetGroupConvLayout(GetSwappedNCLayout(MemLayout_t::NCHW), true);
+    }
+
+    ShaderModel conv(problem, config.n_groups, Winodata, Winofilter);
+
+    BuffInfo d_buf(d_layout,
+                   conv.N,
+                   conv.C,
+                   conv.d_H_window,
+                   conv.d_W_window,
+                   group_cnt,
+                   GetTypeSize(problem.GetInDataType()));
+    BuffInfo o_buf(o_layout,
+                   conv.N,
+                   conv.K,
+                   conv.out_h,
+                   conv.out_w,
+                   group_cnt,
+                   GetTypeSize(problem.GetOutDataType()));
+    BuffInfo f_buf(f_layout,
+                   conv.K,
+                   conv.C,
+                   conv.R,
+                   conv.S,
+                   group_cnt,
+                   GetTypeSize(problem.GetWeightsDataType()));
+
+    const auto d_strides = d_buf.stride;
+    const auto o_strides = o_buf.stride;
+
+    auto f_strides       = f_buf.stride;
+    f_strides.h          = is_forward ? f_strides.h : -f_strides.h;
+    f_strides.w          = is_forward ? f_strides.w : -f_strides.w;
+    uint32_t f_RS_offset = is_forward ? 0 : ((conv.R - 1) * conv.S + (conv.S - 1));
```

Comment:
Filter flip is only needed for Bwd Data direction.

```suggestion
    f_strides.h          = is_forward || is_backWrW ? f_strides.h : -f_strides.h;
    f_strides.w          = is_forward || is_backWrW ? f_strides.w : -f_strides.w;
    uint32_t f_RS_offset = is_forward ? 0 : ((conv.R - 1) * conv.S + (conv.S - 1));
```